vizro-core/docs/pages/user_guides/container.md

```diff
@@ -105,3 +126,173 @@ To add a [`Container`][vizro.models.Container] to your page, do the following:
         [![Container]][Container]
 
     [Container]: ../../assets/user_guides/components/containers.png
+
+
+## Nested containers
+Containers can be nested, providing a hierarchical structure for organizing components.
+This nesting capability allows users to create more complex layouts and manage related components at any level of granularity.
+
+To create nested containers, simply add a `Container` to the `components` argument of another `Container`.
+
+!!! note
+
+    Note that an almost identical layout can also be achieved using [`Page.layout`](layouts.md), see the
+    [advanced grid example](layouts.md#grid-advanced-example) on how this can be done. Here we use `Container` instead
+    of `Page.layout` because we need finer control of selected grid areas (e.g. adding different titles to each subgrid
+    and have different spacing between groups of components).
+
+!!! example "Nested Containers"
+    === "app.py"
+        ```py
+
+        import vizro.models as vm
+        import vizro.plotly.express as px
+        from vizro import Vizro
+
+        df = px.data.gapminder()
+
+        nested_container_one = vm.Container(
+            title="Nested Container I",
+            layout=vm.Layout(grid=[[0, 1], [2, 2]]),
+            components=[
+                vm.Graph(
+                    figure=px.line(
+                        df,
+                        title="Graph 1 - Nested Container I",
```

Comment:
These titles do no appear fully in the screenshot (numbers are missing) which is actually not ideal, because it helps link the code to the outcome. Maybe we can shorten them so that they do?

Reply:
Yeah, it's because the titles do not automatically wrap. I've just shortened them now!